app/controllers/idv/in_person/verify_info_controller.rb

```diff
@@ -57,14 +57,6 @@ def flow_param
         'in_person'
       end
 
-      # state_id_type is hard-coded here because it's required for proofing against
-      # AAMVA. We're sticking with driver's license because most states don't discern
-      # between various ID types and driver's license is the most common one that will
-      # be supported. See also LG-3852 and related findings document.
-      def set_state_id_type
-        pii_from_user[:state_id_type] = 'drivers_license' unless invalid_state?
-      end
```

Comment:
I want to run this by Team Joy. I think I hadn't initially grokked exactly what this was doing.

In **doc auth**, we have `state_id_type` from another vendor's response and can pass that to AAMVA. This, however, is in the **IPP** flow, where data parsed from the image of the driver's license is unavailable, because the user is not taking this path. This code was setting it to `drivers_license`, which is the most common value.

It turns out that, in VerificationRequest, we were never (whether IPP or not) actually sending `state_id_type` to AAMVA, ever. We passed it down the chain thinking we were, but it was (seemingly inadvertently) left out of the XML template.

I think there are 3 options we could take in this controller here:

1. Continue just claiming the document is a driver's license when we don't know, because it's correct the majority of a time.
2. Ask the user, in the UI, if they are using a Driver's License or (non-driver) ID Card, and send that value.
3. Remove this block and do not send `state_id_type` to AAMVA when we don't have that information.

I went for option 3 here, which feels most correct. We know it's OK to not send because that's currently what we do for everyone. Option 2 could be added later if Team Joy wants.

Reply:
FTR I agree we should not send this information to AAMVA unless the user has actually given it to us (or maybe in some states we can suss out the document type based on the format of the ID number? I don't actually know)

Reply:
Sorry so late getting to this. I agree, option 3 seems like the best path. I will talk with product about option 2. No need to implement new behavior in your pr